CMakeLists.txt

```diff
@@ -553,6 +555,32 @@ elseif(CMAKE_SYSTEM_NAME MATCHES "Windows")
 
         SET(CMAKE_C_FLAGS_RELEASE     "/GL /O1b2s")
         SET(CMAKE_C_FLAGS_DEBUG       "/Od")
+
+    elseif(TOOLCHAIN STREQUAL "ARM_CLANG")
+        set(TOOLCHAIN_PATH C:/ArmStudio/sw/ARMCompiler6.18)
```

Comment:
Please don't hardcode it.

I have installed Development Studio 2022.1, then it is at:
C:\Program Files\Arm\Development Studio 2022.1\sw\ARMCompiler6.18

I dont check we need TOOLCHAIN_PATH at all. We can require the user open command prompt with ARMCLANG in the PATH.